### ISSUE TITLE
clean and/or remove cached download files

### ISSUE THREAD

**Issue by maphew** [maphew/apt#14]
<s>Occasionally a file in the downloads cache is corrupted or doesn't match it's mirror equivalent. Currently one has to remove that local file manually, and locating it in order to do so is a pain. We should either automatically overwrite the local file when downloading, or</s> provide a means to clean the cache of that and/or all files.

...might be able to join or match this with removing versions older than X.

**Comment by amr66** [comment]
true, this touches another problem that should be solved: version check. I found the simple one > two in get_new(), which will not work. comparing string will result in true for '1.11.1' < '1.9.1' for example. Solutions exists, e.g.  here: 
http://stackoverflow.com/questions/1714027/version-number-comparison
